brainglobe_atlasapi/atlas_generation/atlas_scripts/demba_dev_mouse.py

```diff
@@ -0,0 +1,345 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+import numpy as np
+import pooch
+from brainglobe_utils.IO.image import load_any
+from rich.progress import track
+from scipy.ndimage import zoom
+
+from brainglobe_atlasapi import BrainGlobeAtlas, utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+@dataclass
+class AtlasMetadata:
+    version: int
+    name: str
+    citation: str
+    species: str
+    atlas_link: tuple[str, str]
+    orientation: str
+    root_id: int
+
+
+# Define the atlas metadata
+METADATA = AtlasMetadata(
+    version=0,
+    name="demba_dev_mouse",
+    citation="https://doi.org/10.1101/2024.06.14.598876",
+    atlas_link="https://doi.org/10.25493/V3AH-HK7",
+    species="Mus musculus",
+    orientation="rsa",
+    root_id=997,
+)
+
+data_file_url = (
+    "https://data.kg.ebrains.eu/zip?container=https://data-proxy.ebrains.eu/api/v1/"
+    "buckets/d-8f1f65bb-44cb-4312-afd4-10f623f929b8?prefix=interpolated_segmentations",
+    "https://data.kg.ebrains.eu/zip?container=https://data-proxy.ebrains.eu/api/v1/"
+    "buckets/d-8f1f65bb-44cb-4312-afd4-10f623f929b8?prefix=interpolated_volumes",
+)
+resolution_to_modalities = {
+    25: ["stpt", "mri", "lsfm", "allen_stpt"],
+    10: ["allen_stpt"],
+    20: ["stpt", "allen_stpt"],
+}
+
+
+def download_resources(download_dir_path, atlas_file_url, atlas_name):
+
+    utils.check_internet_connection()
+
+    download_name = atlas_name
+    destination_path = download_dir_path / download_name
+    for url in atlas_file_url:
+        """
+        Slight issue that the hash seems different each time. I think the
+        files are being zipped on the server each time we request and it's
+        changing the hash somehow (Maybe date and time is encoded in the
+        file when zipped)
+        """
```

Comment:
This is a useful comment and could be moved to the function definition for easier reading.

Reply:
good suggestion